File: 136-single-number/136-single-number.java
class Solution {
public int singleNumber(int[] nums) {

    int i = 0;
    int count = 0;

    Arrays.sort(nums);

    for (i = 0; i < nums.length -1; i++)
    {
        if(nums[i] == nums[i+1])
            count++;
        else
            count--;

        if (count < 0)
            return nums[i];
    }

    return nums[i];
    
}
}

File: 136-single-number/136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = (nums) => {
    
    let sortNums = nums.sort();
    let count = 0;
    for(var i=0; i<nums.length-1; i++){
        if(sortNums[i]==sortNums[i+1]){
            count++;
        }else{
            count--;
            if (count < 0){
            return sortNums[i];
            }
        }

         
    }
    return sortNums[i];
};
